generator/src/main.rs

```diff
@@ -1115,6 +1115,14 @@ impl Parser {
             } else {
                 quote! { #[derive(Copy, Clone, Debug, Default, PartialEq)] }
             };
+            let aliases = self.struct_aliases.iter().map(|(alias, alias_to)| {
+                if alias_to == name {
```

Comment:
style nit: invert condition and return early to reduce rightward drift

Reply:
How would that work with `quote! {}` here?

Reply:
```rs
if alias_to != name {
    return quote! {};
}
quote! { ... }
```